pkg/file/reader.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2022 The Moov Authors
+// Use of this source code is governed by an Apache License
+// license that can be found in the LICENSE file.
+
+package file
+
+import (
+	"bufio"
+	"io"
+	"strings"
+
+	"github.com/moov-io/x12/pkg/util"
+)
+
+type Scanner struct {
+	scan *bufio.Scanner
+}
+
+func (b *Scanner) GetInterChange() string {
```

Comment:
Does ASC X12 call it "InterChange"? Usually it would be written `GetInterchange()` as "interchange" is one word.